source/edge-server.txt

```diff
@@ -26,9 +32,9 @@ Atlas Edge Server
    Edge Server runs on-premises and handles sync between local devices,
    and bi-directional sync between the Edge Server and Atlas.
 
-   Edge Server is currently in Private Preview. Learn more about previewing
-   the 
-   `Atlas Device Sync Edge Server <https://www.mongodb.com/products/platform/atlas-edge-server>`__.
+.. TODO: Add this CTA text with a blog link once marketing provides the link.
```

Comment:
Note for reviewer: marketing won't be publishing the blog article until Thursday, but we need to merge this today for engineering. So this will publish as a commented-out TODO, which won't be visible on the published page, and I've got a separate Jira ticket to address adding this CTA when the article goes live.